FrejaEidClient/src/main/java/com/verisec/frejaeid/client/enums/FrejaResourceEnvironment.java

```diff
@@ -0,0 +1,33 @@
+package com.verisec.frejaeid.client.enums;
+
+/**
+ * FrejaResourceEnvironment that can be used are:
+ * <br> - <b>PRODUCTION</b> - for sending HTTP requests to production
+ * environment
+ * <br> - <b>TEST</b> - for sending HTTP requests to test environment
+ */
+public enum FrejaResourceEnvironment {
```

Comment:
Does it make sense that instead of introducing new enum, we re-use existing enum FrejaEnvironment?
Currently, FrejaEnvironment only contains address of RP service, we could expand it and add property resourceServiceUrl. That way all environment specific things are in one place.

Reply:
Done.